scripts/generate-eth-keypair.py

```
@@ -46,39 +46,23 @@ def main():
     print("3. Store the private key securely in Azure Key Vault")
-    print("4. This wallet has NO FUNDS - you must fund it before use")
+    print("4. Keep this agent wallet unfunded; strategy funds belong in the vault")
     print()
     print("📝 Next Steps:")
     print("-" * 60)
-    print("For TESTNET:")
-    print("  1. Fund this address with testnet tokens")
-    print("  2. Store credentials in Azure Key Vault:")
+    print("Store this agent in the dedicated Key Vault for the intended environment:")
     print()
     print(f"     az keyvault secret set \\")
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)